src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetrySerializer.cs

```diff
@@ -8,8 +8,9 @@
 
     internal class TelemetrySerializer
     {
+        internal readonly Transmitter Transmitter;
```

Comment:
nitpick: Why the capitalization change?  This isn't a property or a constant, does StyleCop want readonly variables to be capitalized also?